tests/nested/manual/preseed/task.yaml

```diff
@@ -41,6 +41,13 @@ prepare: |
   mv snapd-from-deb.snap snapd.snap
   inject_snap_into_seed "$IMAGE_MOUNTPOINT" snapd
 
+  # inject a snap that uses system-usernames into the seed to confirm that works
+  # as expected
+  # TODO: replace this snap with a simpler one instead that is smaller, this one
+  # is 37M, but test-snapd-daemon-user does not have a daemon yet
+  snap download --edge --basename=test-postgres-system-usernames test-postgres-system-usernames
```

Comment:
This snap is not deleted,, I'll create a pr to auto-clean the test dir as we have for the main suite.

Reply:
ah good catch, I hadn't realized this isn't done for nested/manual tests